### PR TITLE
Add collector duration metrics

### PR DESCRIPTION
**Pull request by SuperQ**
Add a last collector durations.
* Remove obsolete total duration metric.
Closes: https://github.com/prometheus/mysqld_exporter/issues/188

### CODE REVIEW

File: mysqld_exporter.go [hunk 1]
```diff
@@ -163,22 +163,17 @@ var landingPage = []byte(`<html>
 // Exporter collects MySQL metrics. It implements prometheus.Collector.
 type Exporter struct {
 	dsn             string
-	duration, error prometheus.Gauge
+	error           prometheus.Gauge
 	totalScrapes    prometheus.Counter
 	scrapeErrors    *prometheus.CounterVec
 	mysqldUp        prometheus.Gauge
+	scrapeDurration *prometheus.GaugeVec
```
Comment (brian-brazil) [inline mysqld_exporter.go]: Duration

Comment (SuperQ) [inline mysqld_exporter.go]: Durr, :)

**Comment by SuperQ** [comment]
@brian-brazil Ok, now with Const metrics.

**Comment by brian-brazil** [comment]
:+1:

**Pull request merged**